src/ITProfessions.ts

```diff
@@ -1,7 +1,22 @@
-export const ITProfessions = [
-  'Backend',
-  'Frontend',
-  'UX-UI',
-  'Devops',
-  'QA',
-  'Project-Manager'];
+export const ITProfessions: { name: string, description: string }[] = [
```

Comment:
It's great that we have type checking here 👏 Maybe we could also extract it to an interface, like so:
```
interface ITProfession {
   name: string;
   description: string;
}
```
and: 
```
export const ITProfessions: ITProfession[] = ...
```
At present, we are inlining it and that's totally fine, but if our interface grows (see my other comment) a separate interface may be more readable :)

Reply:
Done